Reference/google/protobuf/any_test.pb.swift

```diff
@@ -66,20 +66,18 @@ struct ProtobufUnittest_TestAny: SwiftProtobuf.Message, SwiftProtobuf._MessageIm
 
     init() {}
 
-    func copy() -> _StorageClass {
-      let clone = _StorageClass()
-      clone._int32Value = _int32Value
-      clone._anyValue = _anyValue
-      clone._repeatedAnyValue = _repeatedAnyValue
-      return clone
+    init(storage source: _StorageClass) {
```

Comment:
Swift doesn't seem to have any guidance on naming of copy initializers for classes since there aren't any that I could find in stdlib.

I recommended `storage:` because it mirrors the way the `[NSString initWithString:]` copy initializer gets imported into Swift as `init(string:)`, but ¯\\\_(ツ)\_/¯